lib/chat_api_web/controllers/customer_controller.ex

```diff
@@ -36,6 +36,14 @@ defmodule ChatApiWeb.CustomerController do
     end
   end
 
+  def update_metadata(conn, %{"id" => id, "metadata" => metadata}) do
+    customer = Customers.get_customer!(id)
```

Comment:
Should we check that it belongs to the same account for these? Maybe we should have a helper function to check that the logged in user is the same one updating their resources

Reply:
ah this endpoint is public to non-logged in users since we need to send this data over from the widget